integration_tests/tests/apply/about_the_person/equality_and_diversity/complete_religion.cy.ts

```diff
@@ -0,0 +1,63 @@
+//  Feature: Referrer completes 'religion' question page
+//    So that I can complete the 'Equality questions' task
+//    As a referrer
+//    I want to answer questions on the religion page
+//
+//  Scenario: submits a valid answer to religion page
+//    Given I'm on the 'religion' question page
+//    When I give a valid answer
+//    Then I am taken to the task page (temporarily)
+
+import { ReligionPage } from '../../../../pages/apply/about_the_person/equality_and_diversity'
+import TaskListPage from '../../../../pages/apply/taskListPage'
+import Page from '../../../../pages/page'
+import { personFactory, applicationFactory } from '../../../../../server/testutils/factories/index'
+
+context('Visit "About the person" section', () => {
+  const person = personFactory.build({ name: 'Roger Smith' })
+
+  beforeEach(function test() {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+
+    cy.fixture('applicationData.json').then(applicationData => {
+      applicationData['equality-and-diversity-monitoring'] = {}
+      const application = applicationFactory.build({
+        id: 'abc123',
+        person,
+      })
+      application.data = applicationData
+      cy.wrap(application).as('application')
+      cy.wrap(application.data).as('applicationData')
+    })
+  })
+
+  beforeEach(function test() {
+    // And an application exists
+    cy.task('stubApplicationGet', { application: this.application })
+    cy.task('stubApplicationUpdate', { application: this.application })
+
+    // Given I am logged in
+    //---------------------
+    cy.signIn()
+
+    // And I am on the religion page
+    // --------------------------------
+    cy.visit('applications/abc123/tasks/equality-and-diversity-monitoring/pages/religion')
```

Comment:
It might be neater to have a `ReligionPage.visit(this.application)` function?

Like I've been using in the Health section WIP https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/blob/e9b3f332afcb3cad0eaa761b2a65ad98375f4f3a/integration_tests/pages/apply/risks-and-needs/physicalHealthPage.ts#L10-L18

Reply:
ooh cool, yes will use that 👍